spec/support/server_selection.rb

```diff
@@ -149,7 +149,7 @@ def in_latency_window
         #
         # @since 2.0.0
         def candidate_servers
-          @candidate_servers.select { |s| s['type'] != 'Unknown' }
+          @candidate_servers.select { |s| s['type'] != 'Unknown' && s['type'] != 'PossiblePrimary' }
```

Comment:
This is really minor but in terms of style, it might be easier to write it like this:

```ruby
@candidate_servers.select { |s| !['Unknown', 'PossiblePrimary'].include?(s['type'] ) }
```

That way, if we ever need to add a certain server type to exclude, we can just add it to the array.

Reply:
Done